pkg/cmd/codespace/common.go

```diff
@@ -31,23 +31,40 @@ type executable interface {
 	Executable() string
 }
 
+type progressIndicator interface {
+	StartProgressIndicatorWithLabel(s string)
+	StopProgressIndicator()
+}
+
+type liveshareApiClient interface {
+	GetCodespace(ctx context.Context, name string, includeConnection bool) (*api.Codespace, error)
+	StartCodespace(ctx context.Context, name string) error
+}
+
+type statusChecker func(context.Context, progressIndicator, liveshareApiClient, *api.Codespace) error
+type sshClient func(app *App, ctx context.Context, sshArgs []string, opts sshOptions) (err error)
```

Comment:
Same here with context.Context being a second param, should be first.

Reply:
👍🏻  Sorry I missed that